include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
@rhornung67 These are mathematical identities, the identity in a group is supposed to be the thing under an operator such that when any other element is operated on with it, you get the thing you operated on (1 * anything = that thing, 0 + anything = that thing, true AND anything = that thing, false OR anything = that thing). Finally, a part of math matters at the lab that I know